bindata/manifests/operator-webhook/002-rbac.yaml

```diff
@@ -9,6 +9,8 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: operator-webhook
+  labels:
```

Comment:
There are more objects created, do we want to label them also?

Reply:
That is a valid point. In case others will confim that I'll add it in all generated objects. The intention was to be able to monitor non-namespaced objects created by operator